cmtc/src/main/scala/cmt/client/cli/SharedOptions.scala

```diff
@@ -7,7 +7,7 @@ import cmt.client.cli.ArgParsers.studentifiedRepoArgParser
 
 final case class SharedOptions(
     @ExtraName("s")
-    studentifiedRepo: StudentifiedRepo)
+    studentifiedRepo: Option[StudentifiedRepo] = None)
```

Comment:
hmmm.... wasn't sure whether to make the `studentifiedRepo` optional or to make the `shared` property of each `Options` object optional... i'm wondering now i've raised the PR whether it should be the `shared`... WDYT?

Reply:
Probably slightly better to make it shared...